libs/documentation/src/lib/storybook/formly/formly-stepper/formly-stepper-advanced/formly-stepper-advanced.component.html

```diff
@@ -0,0 +1,160 @@
+<button class="usa-button margin-bottom-3" (click)="toggleLinearMode()">
+  Toggle Linear Mode
+</button>
+
+<custom-stepper-demo
+  #stepper
+  [queryParamKey]="'sdsAdvancedStepper'"
+  [model]="model"
+  [stepValidityMap]="stepValidityMap"
+  (stepChange)="onStepChange($event)"
+  [linear]="linear"
+  [validateStepsOnInit]="validateStepsOnInit"
+  (saveData)="onSaveData($event)"
+  (submittedData)="handleSubmit($event)"
+>
+  <sds-step
+    id="welcome"
+    text="Welcome"
+    [stepTemplate]="gatherInformation"
+    [stepValidationFn]="stepMap.welcome.validationFn"
+  >
+  </sds-step>
+
+  <sds-step id="entityDetails" text="Entity Details" [editable]="false">
+    <sds-step id="permission" text="permission" [fieldConfig]="stepMap.permission.fieldConfig">
+      <h1 sdsStepHeader class="margin-top-0">
+        permission
```

Comment:
Capitalize

Reply:
this part was also copied, but i still capitalized it

Reply:
Still lower case as of your last push.

Reply:
I changed the h1 inside the step previously, but now I capitalized the spelling in the step aswell